frontend/packages/console-dynamic-plugin-sdk/src/styles/common.scss

```diff
@@ -0,0 +1,13 @@
+@import './vars.scss';
+
+.dps-m-pane__body {
```

Comment:
@fhlavac how did you derive these styles? Did they come from the console?

Reply:
@florkbr yep, they are in common and present when the console loads. Just thought we want to separate them to the SDK. Is it ok to remain the component dependent on the common styles?